internal/gensnippets/gensnippets.go

```diff
@@ -112,12 +160,14 @@ var skip = map[string]bool{
 	"cloud.google.com/go/longrunning":              true, // Helper.
 	"cloud.google.com/go/monitoring/apiv3":         true, // Has v2.
 	"cloud.google.com/go/translate":                true, // Has newer version.
+	"../..":                                        true,
```

Comment:
I don't think this is a special case for the code, is it?

Reply:
Done